desktop-app/app/components/CreateIssue/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {getEnvironmentInfo} from '../../utils/generalUtils';
+import Button from '@material-ui/core/Button';
+
+const {openNewGitHubIssue} = require('electron-util');
+
+const {
+  appVersion,
+  electronVersion,
+  chromeVersion,
+  nodeVersion,
+  v8Version,
+  osInfo,
+} = getEnvironmentInfo();
+
+const reportBody = state => `Hi I'm reporting an app crash:
+<details>
+<summary>Environment</summary>
+
+- Version: ${appVersion}
+- Electron: ${electronVersion}
+- Chrome: ${chromeVersion}
+- Node.js: ${nodeVersion}
+- V8: ${v8Version}
+- OS: ${osInfo}
+</details>
+
+<details>
+<summary>Error Info</summary>
+${state.error}
```

Comment:
please, left a blank line after `<details>` tag as said in https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab
and also wrap JSON code in ```` ```json  ``` ```` code blocks